ckanext/who_afro/assets/webassets.yml

```diff
@@ -6,6 +6,12 @@ who-afro-theme:
     - css/who-afro.css
     - css/promoted-plus.css
 
+user-profile:
+  filter: cssrewrite
+  output: who-afro/%(version)s-userprofile.css
+  contents:
+    - css/userprofile.css
+
```

Comment:
I'm a bit worried about the inconsistency here?  :thinking: Why create a seperate css file for the user profile pages, when all the other css is bundled together in one?